src/TableOfContents.php

```diff
@@ -502,7 +504,7 @@ public function insertTOC()
 				$toc_orientation = $this->m_TOC[$toc_id]['TOCorientation'];
 				$TOCuseLinking = $this->m_TOC[$toc_id]['TOCuseLinking'];
 				$TOCusePaging = $this->m_TOC[$toc_id]['TOCusePaging'];
-				$toc_bookmarkText = $this->m_TOC[$toc_id]['TOCbookmarkText']; // *BOOKMARKS*
+				$toc_bookmarkText = Arrays::get($this->m_TOC[$toc_id], 'TOCbookmarkText', $this->TOCbookmarkText); // *BOOKMARKS*
```

Comment:
Should the default really be $this->TOCbookmarkText? To preserve current behaviour, NULL or an empty string would be better, wouldn't it?

Reply:
Ah, yes. Agreed. I've updated the PR.